site/installation/migration.md

```diff
@@ -248,15 +257,56 @@ changes to the resources in the live cluster.
    setters are migrated to a new and simple declarative version, package
    consumers can easily declare all the setter values and render them all at
    once.
-5. The `gcr.io/kpt-fn/fix` function doesn’t migrate the `inventory object` to
-   `ResourceGroup`. If you are using the inventory object to manage live
-   resources, please refer to `live migrate` command docs to perform [live
-   migration] separately.
 
-Finally, test your migrated kpt package end-to-end and make sure that the
+Test your migrated kpt package end-to-end and make sure that the
 functionality is as expected. `gcr.io/kpt-fn/fix` is a helper for migration and
 doesn't guarantee functional parity.
 
+Finally, publish your package to git by upgrading the major version so that your
+consumers can fetch with the specific version.
+
+### For Package Consumers
+
+This section walks package consumers through an end to end workflow in order to
+fetch the latest version of the published package which is compatible with kpt `v1.0`
+and migrate the local customizations(if any) already performed to their existing package.
+
+- As a package consumer, you are expected to have some version of the kpt package
+  which is compatible with kpt `v0.39` on your local.
+  Fetch the latest version of published remote package in to a new directory different
+  from your existing package directory.
+
+```shell
+mkdir v1_pkg_dir; cd v1_pkg_dir
+
+# you must be using v1.0 version of kpt to run this command
+$ kpt pkg get https://example.com/some-pkg@latest
+```
+
+- You might have performed some customizations to your existing package such as,
+  updated setter values, made some in-place edits to your resources etc.
+  Please make sure that you capture those customizations and add them to the newly
+  fetched package.
+
+- Render the package resources with customizations
+
+```shell
+$ kpt fn render
+```
+
+- Skip this point if you are not using `kpt live` functionality.
```

Comment:
Rephrase: The following is only applicable if you're using `kpt live`...

Reply:
Done.